specification/securitycenter/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -0,0 +1,1949 @@
+                        },
+                        "sku": {
+                            "type": "string",
+                            "description": "The security solutions' image sku"
+                        }
+                    }
+                },
+                "type": {
+                    "type": "string",
+                    "description": "Type of the resource",
+                    "enum": [
+                        "Microsoft.Security/locations/discoveredSecuritySolution"
+                    ]
+                }
+            },
+            "allOf": [{
+                "$ref": "#/definitions/Resource"
+            }]
+        },
+        "securityFamily": {
+            "type": "string",
+            "description": "The security family of the discovered solution",
+            "enum": [
+                "Waf",
+                "Ngfw",
+                "SaasWaf",
+                "Va"
+            ]
+        },
+        "JitNetworkAccessPoliciesList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/JitNetworkAccessPolicy"
+                    }
+                }
+            }
+        },
+        "JitNetworkAccessPolicy": {
+            "type": "object",
+            "required": [
+                "kind",
+                "properties",
+                "id",
+                "name",
+                "type",
+                "location"
+            ],
+            "properties": {
+                "properties": {
+                    "required": [
+                        "provisioningState",
+                        "virtualMachines"
+                    ],
+                    "type": "object",
+                    "properties": {
+                        "virtualMachines": {
+                            "type": "array",
+                            "description": "Configurations for Microsoft.Compute/virtualMachine resource type.",
+                            "items": {
+                                "$ref": "#/definitions/JitNetworkAccessRequest"
+                            }
+                        },
+                        "requests": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/JitNetworkAccessPolicyVirtualMachine"
+                            }
+                        },
+                        "provisioningState": {
+                            "type": "string",
+                            "description": "Gets the provisioning state of the Just-in-Time policy.",
+                            "enum": [
+                                "Succeeded",
+                                "Failed"
+                            ]
+                        }
+                    }
+                }
+            },

```

Comment:
It should also have 'location' property.

Reply:
Added